data/mods/Aftershock/npcs/classes.json

```
@@ -159,5 +159,71 @@
     "chat": "TALK_WHATELY_GENERIC1",
     "faction": "whately_family",
     "gender": "male"
+  },
+  {
+    "type": "npc",
+    "id": "sadie",
+    "name_unique": "Sadie",
+    "name_suffix": "forgotten experiment",
+    "class": "afs_forgotten_experiment",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "TALK_Sadie_1",
+    "mission_offered": "MISSION_sadie_nuts",
+    "faction": "no_faction",
+    "gender": "female"
+  },
+  {
+    "type": "npc_class",
+    "id": "afs_forgotten_experiment",
+    "name": "forgotten experiment",
+    "job_description": "What a magical world to explore…",
+    "common": false,
+    "traits": [
+      [ "PRETTY", 100 ],
+      [ "OPTIMISTIC", 100 ],
+      [ "INT_DOWN_3", 100 ],
```

```suggestion
      [ "INT_DOWN_3", 100 ],
      [ "FORGETFUL", 100 ],
```
Won't have any real impact on the NPC but thematically it fits.